Cards/CardContainer.gd

```diff
@@ -52,7 +52,9 @@ var _discard_timer: SceneTreeTimer = null
 func _ready() -> void:
 	PhaseManager.on_phase_changed.connect(_on_phase_changed)
 	CardManager.set_card_container(self)
-	CardManager.on_card_action_finished.connect(finish_active_card_action)
+	CardManager.on_card_action_finished.connect(finish_active_card_action.unbind(1))
+	# ? all calls functions connected to on_card_action_finished don't even use the data of the card passed with the signal
+	# ? does the signal even need to pass the card data since no function connected uses it ?
```

Comment:
I'd prefer to keep it just encase we need it at some point

Reply:
fair enough
I didn't remove it directly precisely because i thought it could be useful